EkaterinaUlitina/src/main/java/HomeWorkFromLesson6/MaxInDoubleDimensionalArray.java

```diff
@@ -0,0 +1,23 @@
+package HomeWorkFromLesson6;
+
+import java.util.Random;
+
+public class MaxInDoubleDimensionalArray {
+    public static void main(String[] args) {
+
+        int[][] arr = new int[5][8];
+        Random random = new Random();
+
+        int max = 0;
```

Comment:
What if all random numbers are negative and real maximum is -11 for example? Will it work?

Reply:
Fixed it :) 
Thanx!